package.json

```diff
@@ -222,6 +222,7 @@
     "bootstrap": "5.3.3",
     "bootstrap-icons": "^1.11.3",
     "browserslist": "^4.21.4",
+    "clipboard": "^2.0.11",
```

Comment:
Is it (with `prism` deps) required at root level dependency?

Reply:
I didn't test without it but I believe we have to put all the deps at root level for the angular build

Reply:
But they are integrated as script and not angular component, so not sure :S

Reply:
just tested and I confirm that removing `clipboard` from the monorepo root `package.json` breaks the build
![image](https://github.com/user-attachments/assets/bf8b13a4-5f42-45e6-b604-44cb77544415)